backend/api/Services/MissionRunService.cs

```diff
@@ -352,7 +353,10 @@ MissionStatus missionStatus
             missionRun.Status = missionStatus;
 
             await _context.SaveChangesAsync();
+            await Update(missionRun);
```

Comment:
Why do we perform a `SaveChangesAsync` here and then make a call to `Update` which makes another call to `SaveChangesAsync`?